openedx_webhooks/tasks/github.py

```diff
@@ -75,6 +76,15 @@ def pull_request_opened(self, pull_request, ignore_internal=True, check_contract
         # Bots never need OSPR attention.
         return None, False
 
+    if is_no_jira_org_pull_request(pr):
```

Comment:
The JIRA/no-JIRA decision should be lighter-weight.  Continue with the main flow of the function.

Reply:
Call `github_community_pr_comment` eventually just as before, but with the JIRA argument set to None.